jackson-modules/jackson-exceptions/src/test/java/com/baeldung/mappingexception/JacksonMappingExceptionUnitTest.java

```diff
@@ -48,4 +47,29 @@ public final void givenObjectHasNoAccessorsButHasVisibleFields_whenSerializing_t
         assertThat(dtoAsString, containsString("booleanValue"));
     }
 
+    @Test
+    public void givenJsonWithInvalidList_whenDeserializing_thenThrowException() throws JsonParseException, IOException {
```

Comment:
You can remove both exceptions since they are never thrown.

Reply:
Done